system/lib/update_compiler_rt.py

```diff
@@ -23,6 +23,7 @@
     ('lib', 'lsan'),
     ('lib', 'ubsan'),
     ('lib', 'ubsan_minimal'),
+    ('lib', 'profile'),
```

Comment:
Can you make sure you copy these files from compiler-rt at the same revision as the existing file?

Based on system/lib/compiler-rt/readme.txt it looks like they should be copied from https://github.com/llvm/llvm-project/compare/llvmorg-19.1.6

Reply:
What is required or triggers a creation of emscripten-lib-20 branch?  We will need both compiler-rt source based on 20+ along with version 20+ of `llvm-profdata` and `llvm-cov` for the tests

Reply:
Thanks for the update to llvm 20, this now aligns